src/azure-cli/azure/cli/command_modules/batch/_params.py

```diff
@@ -51,11 +51,17 @@ def load_arguments(self, _):
         c.argument('tags', tags_type, help="Space-separated tags in 'key[=value]' format.")
         c.argument('storage_account', help='The storage account name or resource ID to be used for auto storage.', validator=storage_account_id)
         c.argument('keyvault', help='The KeyVault name or resource ID to be used for an account with a pool allocation mode of \'User Subscription\'.', validator=keyvault_id)
+        c.argument('public_network_access', help="The network access type for accessing Azure Batch account. Values can either be enabled or disabled.")
```

Comment:
You can use `arg_type=get_three_state_flag()` to control the input value

Reply:
That forces having a default in the CLI. None is valid on the wire and manually setting a separate default in the client therefore seems invalid